src/test/java/ch/wisv/areafiftylan/TicketTransferRestIntegrationTest.java

```diff
@@ -199,6 +199,24 @@ public void testDoTransfer_Outsider(){
     }
 
     @Test
+    public void testDuplicateSetupForTransfer() {
+        addTicketTransfer(user.getUsername(), userCleartextPassword);
+
+        SessionData login = login(user.getUsername(), userCleartextPassword);
+
+        //@formatter:off
+        given()
```

Comment:
This can also be replaced by addTicketTransfer. That method returns the response which can then be checked.